### PR TITLE
Rename meal-voucher module to meal-voucher-fr

### PR DESCRIPTION
**Pull request by ozgur00**
## Description
[//]: # (Include a short summary of your changes)
[//]: # (If this is a new feature: attach screenshots or a video if applicable)
[//]: # (If this is a bug fix: include a reproduction path)
Rename module from `meal-voucher` to `meal-voucher-fr` to indicate it is only used for French Meal Vouchers.

## Checklist 
- [x] PR is labelled 
- [x] Changes are tested manually

COAND-693

### CODE REVIEW

File: meal-voucher-fr/api/meal-voucher.api
Comment (jreij) [inline meal-voucher-fr/api/meal-voucher.api:1]: This file should be deleted right?

File: meal-voucher-fr/build.gradle [hunk 1]
```diff
@@ -12,14 +12,14 @@ plugins {
     id 'kotlin-parcelize'
 }
 
-ext.mavenArtifactId = "mealvoucher"
+ext.mavenArtifactId = "meal-voucher-fr"
 ext.mavenArtifactName = "Adyen checkout Meal Voucher component"
```
Comment (jreij) [inline meal-voucher-fr/build.gradle:16]: We should add France to this line and the one under, right?

File: meal-voucher-fr/src/main/java/com/adyen/checkout/mealvoucherfr/internal/ui/model/MealVoucherFRComponentParamsMapper.kt [hunk 2]
```diff
@@ -33,7 +33,7 @@ internal class MealVoucherComponentParamsMapper(
             componentSessionParams,
         )
         val commonComponentParams = commonComponentParamsMapperData.commonComponentParams
-        val mealVoucherConfiguration = checkoutConfiguration.getMealVoucherConfiguration()
+        val mealVoucherConfiguration = checkoutConfiguration.getMealVoucherFRConfiguration()
```
[the remark this answers is not in the document]
Comment (ozgur00) [reply meal-voucher-fr/src/main/java/com/adyen/checkout/mealvoucherfr/internal/ui/model/MealVoucherFRComponentParamsMapper.kt]: I thought I checked everything manually after running some searches but missed this one. Removed it.

File: meal-voucher-fr/build.gradle (hunk 1 above)
Comment (ozgur00) [reply meal-voucher-fr/build.gradle]: Good one, added it.

**Review by jreij — APPROVED** [review]

**Pull request merged**